src/sass/Fabric.scss

```diff
@@ -9,6 +9,7 @@
 @import './Fabric.Utilities.Output';
 @import './Fabric.ZIndex.Variables';
 @import './Fabric.Mixins';
+@import './Fabric.i18n';
```

Comment:
I like the idea of namespacing these, and changing the name to `RTL`. i18n, while fun to say, is not terribly intuitive. I think longer-term we'll want to move partials into a `partials` subfolder, but for now I'll rename it to `_Fabric.Mixins.RTL.scss`.